### ISSUE TITLE
Frontend build script fails on Windows

### ISSUE THREAD

**Issue by nbarbettini** [okta/samples-js-angular-1#18]
From https://github.com/okta/samples-nodejs-express-4/issues/18

I tried this with

```
node --version
v7.9.0
```

on Windows 10. The result:
```
C:\Users\Nate\Documents\code\samples-js-angular-1 [master ≡]> node .\scripts\build-frontend.js
'DIST_OUT' is not recognized as an internal or external command,
operable program or batch file.
child_process.js:524
    throw err;
    ^

Error: Command failed: DIST_OUT=C:\Users\Nate\Documents\code\samples-js-angular-1\dist npm run build
    at checkExecSyncError (child_process.js:481:13)
    at execSync (child_process.js:521:13)
    at Object.<anonymous> (C:\Users\Nate\Documents\code\samples-js-angular-1\scripts\build-frontend.js:23:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:423:7)
```

**Comment by nbarbettini** [comment]
Setting an environment variable on Windows requires the `set VARNAME` syntax.

Since [this code](https://github.com/okta/samples-js-angular-1/blob/master/webpack.config.js#L21) already can find the dist folder without `DIST_OUT`, is it necessary to set this var?

**Comment by rchild-okta** [comment]
Ah! Good point, I'm curious what other things will popup on Windows systems. There is one catch to the code you linked - different samples can use a different out directory, which means that:
- The consuming repo is the one that chooses where to build the sample to
- The webpack.config file lives in the frontend package, and would have to read outside of node_modules (to the config file) to see where it should build to

It feels like an easy fix would be to update this build-frontend.js code to use this module:
https://www.npmjs.com/package/cross-env

**Comment by z017249** [comment]
The fix in #19 does not work. I tried with node version 6 and 10. Now getting a different error.

**Comment by nbarbettini** [comment]
Fixed the issue using cross-env.

@z017249 That sounds like a different issue. If you're still having this problem, can you create a new issue and show us your code? Sorry for the delay!
